README.md

```diff
@@ -142,7 +141,7 @@ To install only a few of the available utilities:
 $ make UTILS='UTILITY_1 UTILITY_2' install
 ```
 
-To install every program with a prefix (e.g. uu-echo uu-cat):
+To install every program with a prefix (e.g., uu-echo uu-cat):
```

Comment:
Is this actually required, or is it just stylistic?

Reply:
Here is a thread on it 

https://english.stackexchange.com/questions/16172/should-i-always-use-a-comma-after-e-g-or-i-e